src/CurveController.sol

```diff
@@ -0,0 +1,243 @@
+/// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "./interface/ISwapRouter.sol";
+import "./interface/IConvexRewards.sol";
+import "./interface/IPositionHandler.sol";
+import {ICurveController} from "./interface/ICurveController.sol";
+
+import "./library/Math.sol";
+
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
```

Comment:
Can we use solmate here instead of openzeppelin

Reply:
pushed changes for these